### DISCUSSION TITLE
Can i use CoastSat with Google Colab?

### DISCUSSION THREAD

**Discussion by claudioasn** [kvos/CoastSat#478] [Q&A] [answered]
Hi, I'm trying to follow the steps for implementing CoastSat, and I'd like to know if it's possible to use it with Google Colab instead of Jupyter Notebook / Anaconda. Thank you.

**Comment by kvos** [comment, marked as answer]
hi @claudioasn, I've never tried to use it in Colab, I guess the code should work the same as long as all the packages are installed in the python environment.

I've heard from a colleague (Floris Calkoen) that Colab is a bit deprecated as it's struggling to keep up with the pace of development of Jupyter.

**Reply by claudioasn** [reply]
Thanks for the reply @kvos ! We are here trying to learn how to work with Python and how to implement CoastSat. We've been working for a long time with research applying DSAS and we're having a hard time applying CoastSat because we don't have that much knowledge yet. We're following the tutorial, but some things don't fit. Is there a more detailed step-by-step video or tutorial? Thanks for your help!

**Reply by kvos** [reply]
hi @claudioasn , the most help you will find is by the users on Github, feel free to post any error you encounter in the Issues page, you will get an answer within a couple of days. The tutorial (jupyter notebook + README) is the most detailed explanation of each step.
